packages/v4/patternfly-docs/content/design-guidelines/components/label-group/label-group.md

```diff
@@ -26,3 +26,40 @@ Label groups should be used when there are multiple labels assigned to a compone
 
 ### Label groups in a description list
 <img src="./img/description-list.png" alt="Description list with labels" width="520"/>
+
+## Variations
+
+### Editable label group
+
+#### Adding a new label to a group, without a group label
```

Comment:
I'm a bit confused about this variation. The title says "adding a new label," but the example seems to be about editing a label. I'd also spell out the word "one" here, as in "...there is a need to edit one or more labels."

Reply:
@mcarrano I have updated the design guidelines doc, added an image and made textual changes to address your questions. I have also pushed the updated in the PR. Let me know if any additional changes are needed.